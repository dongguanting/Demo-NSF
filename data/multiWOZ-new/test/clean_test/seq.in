i 'd like to find a train that leaves monday and arrives by 19:00 .
i need to leave from peterborough .
just one please .
i would like to leave from stansted airport after 11:45 .
i 'd like to depart from stevenage after 19:45 .
i 'm looking for a train leaving on tuesday please .
actually i need to go to london liverpool street from cabridge .
i would like to leave anytime after 20:15 .
i am looking for a train going to cambridge .
i would like to leave broxbourne on tuesday .
i 'd just like to arrive by 16:45 please .
yes i would like a ticket for one .
i 'd like a train to cambridge that arrives by 11:30 .
i 'm departing from ely .
i 'm head to leicester from cambridge on saturday .
i meant to say i want to leave after 20:45 .
i want to arrive by 19:30 please
i am looking for a train departing from london kings cross
yes departing from there heading to cambridge on saturday .
the train should go to cambridge and should arrive by 14:00 .
please book that for 5 people .
i am travelling on monday and need to leave after 16:30 .
i need to arrive by 11:15 .
i need to find a train that arrives at 12:30 on wednesday .
i am departing from birmingham new street and going to cambridge .
yes i need to arrive in cambridge by 12:30 please .
i would like to go on saturday .
i need a train leaving cambridge after 17:30 .
i want to go to stansted airport on saturday .
i am looking for a train from cambridge on monday .
yes i need three seats on the train please .
i am looking for a train that goes to london liverpool street and leaves after 17:15 .
i want to leave on friday .
i am looking for a train that departs from cambridge on thursday .
i need to go to ely after 12:00 .
i will be leaving from cambridge .
i need to go to birmingham new street .
monday is the day i plan to travel
just one ticket please .
i want to leave from london liverpool street .
i will depart on wednesday .
i would like to find a train that is leaving cambridge on monday .
i 'll be leaving from london kings cross and need to leave after 17:15 .
i need to leave cambridge on sunday by train .
i need to find a train that is going to arrive in stevenage by 11:45 .
i need to find a train going to stansted airport and departing on thursday .
i am looking for a train leaving from norwich .
i am looking to go to cambridge on tuesday
i will be departing from norwich on wednesday .
can you please book it for 4 people ?
i 'm going from cambridge to the stansted airport .
i would like to leave after 13:45 .
i only need one ticket today .
i want to get a train to travel from cambridge to bishops stortford .
just one ticket please .
hello i need to arrive at london kings cross no later than 16:45 .
i 'm looking for a train please going to cambridge .
i 'm looking for a train leaving from london liverpool street after 20:00 on friday .
i am looking for a train departing from cambridge .
i am looking for a train on sunday from cambridge .
can you verify that it will arrive in stevenage by 09:00 ?
i need a train departing from cambridge .
i will be going to london liverpool street on sunday .
i 'm looking for a train going to cambridge please .
does that depart from broxbourne station ?
i want to leave on sunday
i would prefer to leave after 13:45 please
can i please get information for a train going from cambridge to broxbourne ?
can you give me the train id and price of the first train that departs after 19:45 ?
i 'm looking for a train from cambridge to the stansted airport .
i need a train to cambridge .
around 08:00 and departing for cambridge .
that should work fine can i get tickets for 6 people ?
i need a train from cambridge to norwich please .
i would like to depart on sunday .
i need to arrive by 12:30 .
i only want one ticket please .
i would like to travel on sunday .
i need to leave after 17:15 .
yes i need it booked for 2 people .
i 'd like a train that can get me from cambridge to birmingham new street on monday by 12:45 .
i am looking for a train that leaves on thursday and arrives by 09:15
i am leaving from cambridge where i am now .
i am looking for a train going to birmingham new street and should leave on thursday
i am departing from cambridge and want the train to arrive by 17:15 .
i would like to leave cambridge and go to london liver pool .
i want to arrive by 15:45 on tuesday .
i 'm looking for a train out of cambridge on saturday .
i want to go to kings lynn .
i need a to find a train departing from stansted airport going to cambridge .
i would like to go this saturday to cambridge .
i 'm looking for a train going to ely please .
it will be for thursday .
i just need to arrive by 14:15 .
i am going to cambridge from broxbourne .
i 'd like it to be for saturday .
i just need to leave by 08:30 .
yes that would be fine please book for 6 people .
i am looking for a train from norwich on monday .
the train should leave after 17:00 and should go to cambridge .
i need to arrive by 13:00 .
i need a booking for 4 people .
i need to travel on wednesday and get there by 17:15 .
i would like to leave cambridge after 09:15 .
i would like to book a train for monday departing from kings lynn and arriving at cambridge at 14:45 .
can you please provide me with the departure time of the train arriving by 14:45 or just before ?
i am looking for a train to london kings cross that departs after 08:15 .
i need help finding a train leaving cambridge after 20:45 .
i need to get to london liverpool street .
i would like to leave on thursday
i need a train on wednesday that leaves after 16:00 .
i am going to kings lynn
it should arrive by 09:45 .
i would like to leave on sunday .
to birmingham new street on wednesday .
i need to get a train out of cambridge .
the train should go to stansted airport and should arrive by 21:30 .
i would like to leave on tuesday .
whichever train goes to stansted airport tuesday from cabridge arriving closest to 21:30 .
i 'd like a train to ely that leaves wednesday .
great can i book that for 8 people and get a reference number please ?
i am looking for a train .
i am looking to go to london kings cross and depart from cambridge .
i 'll be leaving on friday after 12:45 .
i 'd like to leave from broxbourne sometime after 12:30 .
i am trying to find a train leaving after 15:15 to go to broxbourne .
i am looking at traveling on sunday .
can you help get tickets for a train on tuesday ?
i need to go to cambridge from stevenage after 10:45 .
i need a train on wednesday
i 'd like a train to london liverpool street that leaves after 10:00 .
i will be leaving from cambridge on friday .
i 'm looking for a train departing from broxbourne .
i am heading to cambridge .
i will need to arrive by 17:15 .
i will be leaving on wednesday .
does that train arrive in cambridge before 17:15 ?
i need a train to cambridge .
i want to leave from ely on friday .
i need to leave after 09:45 .
i will be leaving saturday morning at 08:45
sorry for the confusion i need to get to peterborough thursday night .
i need to leave after 20:15
just one .
i want to go from norwich to cambridge
i would like to arrive in cambridge by 14:15 .
i need a booking for two passengers .
i 'd like to find a train leaving on friday to cambridge .
i 'm looking for a train from cambridge .
i need to get to stansted airport please .
i 'd like to travel on thursday
i need to leave after 16:15 .
i am looking to book a ticket for 2 people for the tuesday train leaving from cambridge and arriving by 10:30 at broxbourne
i will be traveling on monday and need to arrive by 14:15
i would like to travel on monday after 12:00 .
i want to go to leicester .
i need to go to cambridge on monday .
just one please .
i want to book a train that departs from cambridge and arrives in peterborough by 21:45 .
i need a train from london liverpool street station on wednesday .
i need to arrive by 14:45 .
i want to leave on wednesday .
yes that works great can i book that for 5 people please ?
can i find a train to peterborough departing from cambridge
im going to kings lynn
i would be leaving from cambridge .
the train should leave after 10:15 and should go to peterborough
i need a train from cambridge to ely .
am looking for a train it should arrive by 18:30 and should go to peterborough
i need to leave on sunday from cambridge .
i am looking for train going to cambridge please .
i 'm looking for a train going to cambridge .
i will be leaving from brookshite .
i want to go from london liverpool street to cambridge .
i want to leave after 08:45 please
i just need one ticket please .
could you tell me when the train arrives in london liverpool street ?
i actually need to leave on saturday .
i need a train that leaves this thursday from cambridge to ely .
i need to leave after 17:45 so anytime after that would be great .
i am looking for a train from cambridge to leicester .
i will be leaving on monday .
i 'd like to find a train to peterborough .
can you reserve 6 tickets on the next train after 11:15 ?
i would like to leave after 09:00 on sunday .
i am going to cambridge .
i 'll will be leaving from london liverpool street .
i need on that will leave london after 21:00 .
i need to leave from cambridge and get to kings lynn by 8:00 on saturday .
looking for a train going to cambridge .
i am leaving on sunday and need to arrive by 13:00
i 'm looking for a train going to broxbourne please .
i 'm looking for a train that will be going to london kings cross on wednesday .
i need it to arrive by 18:00 .
please find a train that departs from peterborough and arrives in cambridge by 8:45 .
i would like to leave on saturday from london liverpool street
i am looking for a train schedule that departs from cambridge and should go to ely .
can you book that for 3 people and get me a reference number as well please ?
i need to book a train .
i 'm at bishops stortford and would like to go to cambridge .
i would like to travel this wednesday and need to leave after 08:15 .
would you be able to find me a train leaving on sunday ?
i would like to go on wednesday .
i really must arrive by 18:15 to make my meeting .
i 'll be going from cambridge to broxbourne .
i 'm looking for a train to take 5me to cambridge .
i would like to leave from stevenage on thursday and get to cambridge by 9:15 .
i am leaving leicester and i need to leave by 20:30 .
i need a train on tuesday out of kings lynn
i want to leave on tuesday .
looking for a train tuesday leaving kings lynn after 9:30 to go to cambridge .
i need a train out to london liverpool street on saturday .
i need help finding a train leaving cambridge after 15:30 .
im going to cambridge
i 'll be going to peterborough from cambridge .
yes please book it for 1 person and provide the reference number
i am looking for a train leaving after 11:45
i 'll be leaving stansted airport friday to head toward cambridge .
that would work for me can i get a train for 7 people ?
i would like to go to birmingham new street and the train should leave after 14:15 .
i am looking for a train that goes to cambridge around 10:45
i need to get a train from cambridge to norwich .
are there any trains leaving from cambridge on thursday ?
i 'm heading to broxbourne .
i am looking for one that will arrive by 15:00
i need that for 6 people .
i need a train to peterborough from cambridge on sunday .
i would like to leave after 18:15 please .
i will be departing from norwich .
i would like to leave after 14:30 .
i need to book a train departing from cambridge after 9:15 .
i am looking for a train departing from london liverpool street on thursday to go to cambridge .
i need a train to cambridge .
i am looking for a train from kings lynn .
i would like to travel on thursday .
i 'd like to go after 10:00 .
i am looking for a train .
i need to leave on thursday from cambridge .
i am looking for a train to peterborough .
i am looking to go from cambridge to peterborough and leave after 14:15
i 'll be traveling on tuesday .
i need a train going from stevenage to cambridge .
i would like to leave on tuesday and need to arrive by 09:00 .
i want to leave for cambridge by train on wednesday .
i 'll be coming from london kings cross
i am looking for a train departing from london kings cross .
i would like to arrive in cambridge on tuesday by 17:00 .
i 'm leaving london liverpool street on sunday after 20:30 .
there are none closer to a 20:30 departure ?
i need to leave sunday to go to ely .
yes can i get a ticket for 8 people please
i need 2 please .
does that train leave on friday ?
i need a train going from ely to cambridge please .
i 'd like to leave on friday and arrive by 14:45 .
i 'm going from stevenage to cambridge .
yes i would like to book it for 8 people and i will need the reference number .
i want to leave from stevenage on saturday .
i would like to leave after 12:15 .
i will depart any time so long as the train arrives by 20:45 .
i need to make a train reservation for friday and i need it to arrive by 19:00
i will be departing from cambridge .
i will be going to stansted airport .
the train should go to cambridge and should depart from london liverpool street .
i need a train to take me from cambridge to broxbourne .
i will be travelling on monday .
can i get a train from cambridge to bishops stortford ?
on thursday after 20:15 .
i am looking for a train that departs cambridge after 21:00 on thursday .
i am looking for a train to cambridge on saturday .
i would like to travel on saturday .
i 'd like to arrive by 13:45 .
i need a train that leaves on wednesday with an arrival of 12:30 .
i 'm traveling from cambridge to the london liverpool street .
i 'd like to find a train tuesday departing from leicester .
i would like to leave after 11:15 .
yes i would like to go from leicester to cambridge and leave after 11:15 please .
i 'd like to find a train from leiceter on saturday .
i am going to cambridge and need to leave after 12:00
i am looking for a train from london kings cross and leaves on wednesday
i need a train leaving from london liverpool street on friday .
i need to arrive by 14:00
i 'm looking for a train from norwich to cambridge .
i would like to leave on tuesday and arrive by 13:30 .
i am looking for a train on friday that leaves after 21:45 .
i 'll be leaving from cambridge and i want to go to stansted airport .
i need to leave after 21:45 .
i will be departing from london kings cross after 19:00 .
i need it for 6 people .
i am looking for a train leaving london kings cross and arriving by 14:30 .
i 'm going to cambridge
yes please book me something around 10 for 7 people
i will need to leave from cambridge .
i would like to travel on a thursday .
i just need to arrive by 17:45
i would like to catch a train leaving cambridge on sunday .
i need to get to leicester on sunday .
i want to leave after 10:15
i am looking for a train that leaves bishops stortford and goes to cambridge
i plan to leave this saturday .
i would like to arrive in cambridge by 8:15 .
i need a train from london kings cross to cambridge .
is there a train schedule for sunday ?
i 'm looking for a train departing from stevenage .
i need to go to cambridge on friday .
i want to leave sometime after 13:30 .
