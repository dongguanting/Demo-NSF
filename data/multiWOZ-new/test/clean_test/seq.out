O O O O O O O O O B-day O O O B-arrive O
O O O O O B-depart O
O B-people O O
O O O O O O B-depart I-depart O B-leave O
O O O O O O B-depart O B-leave O
O O O O O O O O B-day O O
O O O O O O B-depart I-depart I-depart O O O
O O O O O O O B-leave O
O O O O O O O O B-dest O
O O O O O B-depart O B-day O
O O O O O O O B-arrive O O
O O O O O O O B-people O
O O O O O O B-dest O O O B-arrive O
O O O O B-depart O
O O O O B-dest O B-depart O B-day O
O O O O O O O O O B-leave O
O O O O O B-arrive O
O O O O O O O O B-depart I-depart I-depart
O O O O O O B-dest O B-day O
O O O O O B-dest O O O O B-arrive O
O O O O B-people O O
O O O O B-day O O O O O B-leave O
O O O O O B-arrive O
O O O O O O O O O B-arrive O B-day O
O O O O B-depart I-depart I-depart O O O B-dest O
O O O O O O B-dest O B-arrive O O
O O O O O O B-day O
O O O O O B-depart O B-leave O
O O O O O B-dest I-dest O B-day O
O O O O O O O B-depart O B-day O
O O O B-people O O O O O O
O O O O O O O O O B-dest I-dest I-dest O O O B-leave O
O O O O O B-day O
O O O O O O O O O B-depart O B-day O
O O O O O B-dest O O O
O O O O O B-depart O
O O O O O B-dest I-dest I-dest O
B-day O O O O O O O
O B-people O O O
O O O O O B-depart I-depart I-depart O
O O O O B-day O
O O O O O O O O O O B-depart O B-day O
O O O O O B-dest I-dest I-dest O O O O O B-leave O
O O O O B-depart O B-day O O O
O O O O O O O O O O O O B-dest O B-arrive O
O O O O O O O O B-dest I-dest O O O B-day O
O O O O O O O O B-depart O
O O O O O O B-dest O B-day
O O O O O B-depart O B-day O
O O O O O O B-people O O
O O O O B-depart O O B-dest I-dest O
O O O O O O B-leave O
O O O B-people O O O
O O O O O O O O O B-depart O B-dest I-dest O
O B-people O O O
O O O O O O B-dest I-dest I-dest O O O B-arrive O
O O O O O O O O O B-dest O
O O O O O O O O B-depart I-depart I-depart O B-leave O B-day O
O O O O O O O O B-depart O
O O O O O O O B-day O B-depart O
O O O O O O O O B-dest O O O
O O O O O O B-depart O
O O O O O B-dest I-dest I-dest O O O
O O O O O O O O B-dest O O
O O O O B-depart O O
O O O O O B-day
O O O O O B-leave I-leave O
O O O O O O O O O O B-depart O B-dest O
O O O O O O O O O O O O O O O O B-leave O
O O O O O O O B-depart O O B-dest I-dest O
O O O O O B-dest O
O B-leave O O O B-dest O
O O O O O O O O O B-people O O
O O O O O B-depart O B-dest O O
O O O O O O B-day O
O O O O O B-arrive O
O O O B-people O O O
O O O O O O B-day O
O O O O O B-leave O
O O O O O O B-people O O 
O O O O O O O O O O B-depart O B-dest I-dest I-dest O B-day O B-arrive O
O O O O O O O O O B-day O O O B-arrive
O O O O B-depart O O O O O
O O O O O O O O B-dest I-dest I-dest O O O O B-day
O O O O B-depart O O O O O O O B-arrive O
O O O O O B-depart O O O O O O O
O O O O O O O B-day O
O O O O O O O O B-depart O B-day O
O O O O O B-dest I-dest O
O O O O O O O O O B-depart I-depart O O B-dest O
O O O O O O B-day O B-dest O
O O O O O O O O B-dest O O
O O O O B-day O
O O O O O O B-arrive O
O O O O B-dest O B-depart O
O O O O O O O B-day O
O O O O O O B-leave O
O O O O O O O O B-people O O
O O O O O O O B-depart O B-day O
O O O O O B-leave O O O O B-dest O
O O O O O B-arrive O
O O O O O B-people O O
O O O O O B-day O O O O B-arrive O
O O O O O B-depart O B-leave O
O O O O O O O O B-day O O B-depart I-depart O O O B-dest O B-arrive O
O O O O O O O O O O O O O O B-arrive O O O O
O O O O O O O B-dest I-dest I-dest O O O B-leave O
O O O O O O O B-depart O B-leave O
O O O O O B-dest I-dest I-dest O
O O O O O O B-day
O O O O O B-day O O B-leave I-leave O
O O O O B-dest I-dest
O O O O B-arrive O
O O O O O O B-day O
O B-dest I-dest I-dest O B-day O
O O O O O O O O B-depart O
O O O O O B-dest I-dest O O O O B-arrive O
O O O O O O B-day O
O O O O B-dest I-dest B-day O O O O O B-arrive O
O O O O O O B-dest O O B-day O
O O O O O O B-people O O O O O O O O
O O O O O B-depart O
O O O O O O B-dest I-dest I-dest O O O B-depart O
O O O O O B-day O B-leave O
O O O O O O B-depart O O B-leave O
O O O O O O O O O B-leave O O O B-dest O
O O O O O O B-day O
O O O O O O O O O B-day O
O O O O O B-dest O B-depart O B-leave O
O O O O O B-day
O O O O O O B-dest I-dest I-dest O O O B-leave O
O O O O O B-depart O B-day O
O O O O O O O O B-depart O
O O O O B-dest O
O O O O O O B-arrive O
O O O O O B-day O
O O O O O B-dest O B-arrive O
O O O O O B-dest O
O O O O O B-depart O B-day O
O O O O O B-leave O
O O O O B-day O O B-leave
O O O O O O O O O B-dest B-day O O
O O O O O B-leave
O B-people O
O O O O O B-depart O B-dest
O O O O O O O O B-arrive O
O O O O O B-people O O
O O O O O O O O O B-day O B-dest O
O O O O O O O B-depart O
O O O O O B-dest I-dest O O
O O O O O O B-day
O O O O O B-leave O
O O O O O O O O B-people O O O B-day O O O B-depart O O O B-arrive O B-dest
O O O O O B-day O O O O O B-arrive
O O O O O O B-day O B-leave O
O O O O O B-dest O
O O O O O B-dest O B-day O
O B-people O O
O O O O O O O O O B-depart O O O B-dest O B-arrive O
O O O O O B-depart I-depart I-depart O O B-day O
O O O O O B-arrive O
O O O O O B-day O
O O O O O O O O O B-people O O O
O O O O O O B-dest O O B-depart
O O O B-dest I-dest
O O O O O B-depart O
O O O O O B-leave O O O O B-dest
O O O O O B-depart O B-dest O
O O O O O O O O O B-arrive O O O O B-dest
O O O O O B-day O B-depart O
O O O O O O O B-dest O O
O O O O O O O O B-dest O
O O O O O B-depart O
O O O O O B-depart I-depart I-depart O B-dest O
O O O O O B-leave O
O O O B-people O O O
O O O O O O O O O B-dest I-dest I-dest O
O O O O O O B-day O
O O O O O O O B-day O B-depart O B-dest O
O O O O O B-leave O O O O O O O O
O O O O O O O B-depart O B-dest O
O O O O O B-day O
O O O O O O O O B-dest O
O O O B-people O O O O O O B-leave O
O O O O O B-leave I-leave O B-day O
O O O O B-dest O
O O O O O O B-depart I-depart I-depart O
O O O O O O O O B-leave O
O O O O O B-depart O O O B-dest I-dest O O O B-day O
O O O O O O B-depart O
O O O O B-day O O O O O B-arrive
O O O O O O O O B-dest O O
O O O O O O O O O O O B-dest I-dest I-dest O B-day O
O O O O O O B-arrive O
O O O O O O O B-depart O O O B-dest O B-arrive O
O O O O O O B-day O O O O
O O O O O O O O O O B-depart O O O O B-dest O
O O O O O B-people O O O O O O O O O O O
O O O O O B-dest O
O O O B-depart I-depart O O O O O O B-dest O
O O O O O O B-day O O O O O B-leave O
O O O O O O O O O O O B-day O
O O O O O O B-day O
O O O O O B-arrive O O O O O
O O O O O B-depart O B-dest O
O O O O O O O O O O B-dest O
O O O O O O B-depart O B-day O O O O O O O
O O O B-depart O O O O O O B-leave O
O O O O O B-day O O B-depart I-depart
O O O O O B-day O
O O O O B-day O B-depart I-depart O O O O O B-dest O
O O O O O O B-dest I-dest I-dest O B-day O
O O O O O O O B-depart O B-leave O
O O O B-dest
O O O O O B-dest O B-depart O
O O O O O B-people O O O O O O
O O O O O O O O B-leave
O O O O B-depart I-depart B-day O O O B-dest O
O O O O O O O O O O O B-people O O
O O O O O O B-dest I-dest I-dest O O O O O O B-leave O
O O O O O O O O O B-dest O B-arrive
O O O O O O O B-depart O B-dest O
O O O O O O B-depart O B-day O
O O O O B-dest O
O O O O O O O O O B-arrive
O O O O B-people O O
O O O O O B-dest O B-depart O B-day O
O O O O O O B-leave O O
O O O O O B-depart O
O O O O O O B-leave O
O O O O O O O O B-depart O O O
O O O O O O O O B-depart I-depart I-depart O B-day O O O B-dest O
O O O O O B-dest O
O O O O O O O B-depart I-depart O
O O O O O O B-day O
O O O O O O B-leave O
O O O O O B-depart O
O O O O O B-day O B-depart O
O O O O O O O B-dest O
O O O O O O B-depart O O O O O B-leave
O O O O O B-day O
O O O O O O B-depart O B-dest O
O O O O O O B-day O O O O O B-arrive O
O O O O O B-dest O O O B-day O
O O O O O B-depart I-depart I-depart
O O O O O O O O B-depart I-depart I-depart O
O O O O O O B-dest O B-day O B-arrive O
O O O B-depart I-depart I-depart O B-day O B-leave O
O O O O O O B-leave O O
O O O O B-day O O O B-dest O
O O O O O O O B-people O O
O O B-people O O
O O O O O B-day O
O O O O O O B-depart O B-dest O O
O O O O O O B-day O O O B-arrive O
O O O O B-depart O B-dest O
O O O O O O O O B-people O O O O O O O O O
O O O O O B-depart O B-day O
O O O O O O B-leave O
O O O B-leave O O O O O O O O B-arrive O
O O O O O O O O B-day O O O O O O O B-arrive
O O O O O B-depart O
O O O O O B-dest I-dest O
O O O O O B-dest O O O O B-depart I-depart I-depart O
O O O O O O O O B-depart O B-dest O
O O O O O B-day O
O O O O O O B-depart O B-dest I-dest O
O B-day O B-leave O
O O O O O O O O B-depart O B-leave O B-day O
O O O O O O O B-dest O B-day O
O O O O O O B-day O
O O O O O O B-arrive O
O O O O O O O B-day O O O O B-arrive O
O O O O B-depart O O B-dest I-dest I-dest O
O O O O O O O B-day O O B-depart O
O O O O O O B-leave O
O O O O O O O O O B-dest O O O O O O
O O O O O O O O O O B-day O
O O O O B-dest O O O O O B-leave
O O O O O O O B-depart I-depart I-depart O O O B-day
O O O O O O B-depart I-depart I-depart O B-day O
O O O O O B-arrive
O O O O O O O B-depart O B-dest O
O O O O O O B-day O O O B-arrive O
O O O O O O O B-day O O O B-leave O
O O O O O B-depart O O O O O O B-dest I-dest O
O O O O O B-leave O
O O O O O B-depart I-depart I-depart O B-leave O
O O O O B-people O O
O O O O O O O B-depart I-depart I-depart O O O B-arrive O
O O O O B-dest
O O O O O O B-leave O B-people O
O O O O O O B-depart O
O O O O O O O B-day O
O O O O O O B-arrive
O O O O O O O O B-depart O B-day O
O O O O O B-dest O O O
O O O O O B-leave
O O O O O O O O B-depart I-depart O O O B-dest
O O O O O B-day O
O O O O O O O O B-arrive O
O O O O O B-depart I-depart I-depart O B-dest O
O O O O O O B-day O
O O O O O O O O B-depart O
O O O O O B-dest O B-day O
O O O O O O B-leave O
